va-gov/includes/veteran-banner.html

```diff
@@ -1,3 +1,7 @@
 <div class="veteran-banner">
-  <div class="veteran-banner-image"></div>
+  <picture class="veteran-banner-image">
```

Comment:
This is supported in all of our supported browsers except for IE 11, but since there's a fallback img element, it displays the desktop image. I think that's a reasonable compromise for a browser that is mostly going to be viewed at desktop screen sizes.